ament_cmake_gtest_isolated/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(ament_cmake_gtest_isolated)
+
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_test_isolated REQUIRED)
```

Comment:
Couldn't this `find_package` be in `ament_cmake_gtest_isolated/ament_cmake_gtest_isolated-extras.cmake` and be a `buildtool_export_depend` too?